src/content/docs/en/pages/devtools/cli/azion-cli-alpha/login/login.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Azion CLI login
+description: Log in to your Azion account and save a Personal Token locally to authorize CLI commands.
+meta_tags: cli, login, authentication
+namespace: documentation_cli_login
+menu_namespace: cliMenuAlpha
+permalink: /documentation/products/cli/login/
+---
+
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case one or more required fields aren't informed through the specific flags, an interactive prompt asks for the missing information.
```